kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -138,6 +138,14 @@ window.ExerciseView = Backbone.View.extend({
 
         // Catch the "next question" button click event -- needs to be explicit (not in "events")
         this.$("#next-question-button").click(this.next_question_clicked);
+        
+        // Workaround so that pressing enter on perseus exercises submits the answer 
```

Comment:
Would be better to trigger this using Backbone event binding:
http://backbonejs.org/#View

(This way it will get cleaned up properly by backbone and we won't have zomebie listeners).